library/src/scala/tasty/util/ShowSourceCode.scala

```diff
@@ -619,15 +623,24 @@ class ShowSourceCode[T <: Tasty with Singleton](tasty0: T) extends Show[T](tasty
       this
     }
 
-    def printArgDef(arg: ValDef): Unit = {
+    def printParamDef(arg: ValDef): Unit = {
       val ValDef(name, tpt, rhs) = arg
       arg.owner match {
         case DefDef("<init>", _, _, _, _) =>
           val ClassDef(_, _, _, _, body) = arg.owner.owner
           body.collectFirst {
             case vdef @ ValDef(`name`, _, _) if vdef.flags.isParamAccessor =>
-              if (!vdef.flags.isLocal && !vdef.flags.isCaseAcessor)
-                this += "val " // TODO `var`s
+              if (!vdef.flags.isLocal) {
+                var printedPrefix = false
+                if (vdef.flags.isOverride) {
+                  this += "override "
+                  printedPrefix = true
```

Comment:
Why skip `val ` (line 641) if we have `override`?

Reply:
We skip `val` if we have NOT `override`